usr/bin/regolith-session-init

```diff
@@ -5,28 +5,47 @@ set -Eeu -o pipefail
 
 source /usr/lib/regolith/regolith-session-common.sh
 
-# --------- Script Main -----------
-
-load_standard_xres  # ~/.Xresources
-load_regolith_xres  # ~/.config/regolith2/Xresources
-
-# Register with gnome-session so that it does not kill the whole session thinking it is dead.
-# See https://zork.net/~st/jottings/gnome-i3.html for details
-if [ -n "$DESKTOP_AUTOSTART_ID" ]; then
-    dbus-send --print-reply --session --dest=org.gnome.SessionManager "/org/gnome/SessionManager" org.gnome.SessionManager.RegisterClient "string:Regolith" "string:$DESKTOP_AUTOSTART_ID"
-fi
+start_i3 () {
```

Comment:
I'm fine to merge this as is but wonder if you considered breaking out the wayland specific login into a separate file.  IMO this would be a better approach as it scales better.  EG if I make a change to this file as-is I may have to test both x11 and sway, whereas if they were discrete files I wouldn't.  Is there a reason why breaking the logic out is problematic?

Reply:
I thought it would be convenient to have the same starter script for both sway and i3. But on second thought, what you're saying does make a lot of sense.